### DISCUSSION TITLE
Welcome to yaya-world Discussions!

### DISCUSSION THREAD

**Discussion by yayoi-chi** [jjon-saxton/yaya-world#4] [Announcements]
Welcome everyone! You can discuss anything you like here. If you have a feature or a page you'd like added to the site, or whatever, you can talk about it here. I'll create issues if we need them. It would be helpful though to report problems through the issues section or as a comment on the page itself as this also adds an issue. This would be anything from corrections (grammar, spelling, wrong information) to broken links.

Thank you and have fun discussing thing!

**Comment by jjon-saxton** [comment]
Cool, we can now look into using these to talk about things! Let's go!

**Reply by yayoi-chi** [reply]
Let's go!
